mappings/net/minecraft/entity/damage/DamageSource.mapping

```diff
@@ -5,7 +5,12 @@ CLASS net/minecraft/class_1282 net/minecraft/entity/damage/DamageSource
 	FIELD field_28100 fallingBlock Z
 	FIELD field_28399 fromFalling Z
 	FIELD field_28400 STALAGMITE Lnet/minecraft/class_1282;
-	FIELD field_34004 noRetaliation Z
+	FIELD field_34004 neutral Z
+		COMMENT Whether the attacked entity should be neutral to the attacker.
+		COMMENT
+		COMMENT @implNote This is used by goats to prevent rammed mobs from retaliating.
```

Comment:
```suggestion
		COMMENT @apiNote This is used by goats to prevent rammed mobs from retaliating.
```
This is about how this is used than how this field is implemented. implnote is usually for method body's code or the field initializer code.